### ISSUE TITLE
Ember data is still broken

### ISSUE THREAD

**Issue by moogus** [emberjs/data#5034]
Hi

Just tried the lastest version of ember data as it's supposed to fix the`ember-data/-private` sorry to say it seems to still not work.  I have this new issue:  
```js
Uncaught Error: Could not find module `ember-data/-private/system/model/internal-model` imported from `ember-data-model-fragments/ext`
```

![image](https://user-images.githubusercontent.com/4889659/27471104-882f92ea-57ee-11e7-9b31-aba7bd1c7513.png)

My dependencies are as follows:
```json
"devDependencies": {
    "babel-eslint": "^7.1.1",
    "broccoli-asset-rev": "^2.4.5",
    "ember-ajax": "^2.4.1",
    "ember-browserify": "1.1.13",
    "ember-changeset": "1.2.0",
    "ember-changeset-validations": "1.2.4",
    "ember-cli": "2.10.0",
    "ember-cli-app-version": "^2.0.0",
    "ember-cli-babel": "^5.1.7",
    "ember-cli-dependency-checker": "^1.3.0",
    "ember-cli-deploy": "1.0.0-beta.2",
    "ember-cli-deploy-build": "0.1.1",
    "ember-cli-deploy-display-revisions": "0.2.2",
    "ember-cli-deploy-revision-data": "0.3.3",
    "ember-cli-emblem": "0.9.0",
    "ember-cli-eslint": "^3.0.0",
    "ember-cli-fastclick": "^1.3.0",
    "ember-cli-htmlbars": "^1.0.10",
    "ember-cli-htmlbars-inline-precompile": "^0.3.3",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-less": "1.5.3",
    "ember-cli-mirage": "0.2.4",
    "ember-cli-mirage-nested": "^1.0.10",
    "ember-cli-moment-shim": "^3.3.1",
    "ember-cli-page-object": "1.10.0",
    "ember-cli-qunit": "^3.0.1",
    "ember-cli-release": "^0.2.9",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-test-loader": "^1.1.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-cli-version-checker": "^1.2.0",
    "ember-cli-yadda": "0.2.1",
    "ember-component-css": "0.3.0",
    "ember-composability": "^0.2.1",
    "ember-cookies": "0.0.13",
    "ember-data": "2.14.2",
    "ember-data-change-tracker": "0.5.2",
    "ember-data-has-many-through": "^1.0.5",
    "ember-data-model-fragments": "2.11.5",
    "ember-elsewhere": "0.4.1",
    "ember-export-application-global": "^1.0.5",
    "ember-gestures": "^0.4.4",
    "ember-hammertime": "1.2.1",
    "ember-hypersearch": "^0.2.0",
    "ember-inflector": "2.0.1",
    "ember-load-initializers": "^0.5.1",
    "ember-metrics": "0.8.1",
    "ember-moment": "^7.0.1",
    "ember-resolver": "^2.0.3",
    "ember-route-action-helper": "2.0.1",
    "ember-validations": "^2.0.0-alpha.4",
    "eslint": "^3.12.2",
    "hammerjs": "^2.0.8",
    "liquid-fire": "0.27.0",
    "loader.js": "~4.0.10"
}
```

**Comment by rwjblue** [comment]
This error is due to ember-data-model-fragments importing things from a private module that is no longer present. 

/cc @workmanw

**Comment by rwjblue** [comment]
You can track status of that addons upgrade efforts over in https://github.com/lytics/ember-data-model-fragments/issues/266

**Comment by workmanw** [comment]
@rwjblue Thanks! @moogus You can see us over on that issue. I had been waiting on ember 2.14 to release before shipping the next version of model-fragments. However at this point, we should probably release a new version of ember-data-model-fragments.
